docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/tag-template-guide/index.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: "Tag guide"
+title: "Snowplow v3"
 date: "2021-11-18"
-sidebar_position: 200
+sidebar_position: 100
 ---
 
-This template implements the Snowplow JavaScript tracker v3. The template supports all the features of the tracker, with a few exceptions due to the limitations of custom templates’ sandboxed JavaScript.
+This template implements the Snowplow JavaScript tracker v3.
+
+The template supports all the features of the tracker, with a few exceptions due to the limitations of custom templates’ sandboxed JavaScript.
```

Comment:
Maybe link to https://developers.google.com/tag-platform/tag-manager/templates/sandboxed-javascript ? Many users will not have experience writing templates so won't have this context.